### PR TITLE
merging main to prod

### PR DESCRIPTION
**Pull request by chxwcodes**
Read over my code plz n thnx

### CODE REVIEW

File: src/Results.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+                    disabled={isSubmitted}
+                    maxLength={30}
+                    required
+                />
+                <button disabled={isSubmitted}>Submit</button>
+            </form>
+    
+            <div className="resultsNav">
+                {isSubmitted ? <Link to='/scoreboard' onClick={handleReturn}>Scoreboard</Link> : null}
+                <button className='returnBtn' onClick={handleReturn}>Back to game</button>
```
Comment (DanRamsauer) [inline src/Results.js:108]: maybe when the user clicks back to game to go to the main page to select the category and difficulty you could pass state of the form into handleReturn and clear out the category and difficulty inputs

File: src/Scoreboard.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        })
+    }, [])
+
+    return (
+        <section className="scoreboard">
+            <div className="wrapper">                                        
+
+                <section className="scoreData">
+                    <h2>Score<span>board</span></h2>
+                    <p>Results are shown by most recent to oldest.</p>
```
Comment (DanRamsauer) [inline src/Scoreboard.js:50]: I like the most recent but I think it would be cool to display it with the highest score on top

**Review by DanRamsauer — APPROVED** [review]
Everything looks great amazing job! Just added 2 comments

**Comment by chxwcodes** [comment]
will be implementing the additional features in a separate commit since they are fairly complicated to do over github UI

resetting of user dropdown values probably will not do since I want to avoid passing props to grandchildren

**Pull request merged**